pkg/controller/clusterdeployment/clusterdeployment_controller_test.go

```diff
@@ -1503,6 +1571,28 @@ func testFailedProvisionTime(time time.Time) *hivev1.ClusterProvision {
 	return provision
 }
 
+func testDeleteOldFailedProvisions(time time.Time, attempt int) *hivev1.ClusterProvision {
```

Comment:
Rename the function to `testOldFailedProvision` since it is building a single old, failed provision. The name of the function as is states that the function is going to test deleting old, failed provisions.